web/static/style.css

```diff
@@ -111,3 +111,12 @@ here by 4 rather than by 2.
     height: 70px;
     object-fit: contain;
 }
+
+.alert {
+    visibility: hidden;
+    display: inline-block;
+}
```

Comment:
Avoid modifying bootstrap's original styles. This would interfere with any unrelated alerts, if we were to add more. Furthermore, Bootstrap has the `inisible` class you can add to the element instead of writing CSS. Similarily, it also has `d-inline-block`.

Not saying writing CSS is bad though. Somtimes it's nicer to have it in a CSS file than to pile 20 classes onto an element in HTML.